dz-13/src/test/java/DbTest.java

```diff
@@ -0,0 +1,74 @@
+import org.junit.Test;
+
+import java.sql.*;
+
+public class DbTest {
+    @Test
+    public void test (){
+        String url = String.format(
+                "jdbc:mysql://%s:%s/%s",
+                "localhost",
+                "3306",
+                "opencart-db"
+        );
+        String username = "opencart";
+        String password = "opencart";
+        try {
+            Connection connection = DriverManager.getConnection(url, username, password);
+
+            ///////-INSERT-/////
+            String insertQuery = "INSERT INTO homework_user_data (first_name, last_name) VALUES (?, ?)";
+            PreparedStatement insertStatement = connection.prepareStatement(insertQuery);
+            insertStatement.setString(1, "Sammie");
+            insertStatement.setString(2, "Smith");
+            insertStatement.executeUpdate();
+            insertStatement.close();
```

Comment:
Use `try/catch` for each query. In this case you will see all errors simultaneously and in catch block use `softAssert.fail()`